src/api/decoding-scanned-codes.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+parent: API Reference
+title: Decoding Scanned Codes
```

Comment:
I'm unsure about this title now - `Decoding` is the action you want to perform with a scanned code, perhaps `Scanned codes` is a better title?

Reply:
+1, this could be the title of a section on this page but I don't think it's the entire page.

Reply:
Done :)

Reply:
LGTM